recipes/drmp3/all/CMakeLists.txt

```diff
@@ -1,34 +1,33 @@
 cmake_minimum_required(VERSION 3.4)
-project(dr_mp3 C)
+project(dr_mp3 LANGUAGES C)
 
-include(conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
+include(GNUInstallDirs)
 
 option(NO_SIMD "Build with define DR_MP3_NO_SIMD" OFF)
 option(NO_STDIO "Build with define DR_MP3_NO_STDIO" OFF)
 
-add_library(${CMAKE_PROJECT_NAME} dr_mp3.c)
```

Comment:
KEEP CMAKE_PROJECT_NAME or PROJECT_NAME at least.

Reply:
Alright, reverted in 69e56587565679ac9304433028d196fb2e74da11. I changed it because I saw it changed like that in other recipes that were updated to v2.